bin/start

```diff
@@ -54,18 +56,24 @@ done
 
 export RUN_IN_DOCKER
 export CONJUR_DEPLOYMENT
-
-source test/bootstrap.env
+export DEV
+export ENV_RELOAD
```

Comment:
why do we need to export this?

Reply:
That is how we did it with the other variables so I followed suit but i can remove that